### ISSUE TITLE
How to send a packet to the server from a client?

### ISSUE THREAD

**Issue by CalvinDKL** [Barbosik/MultiOgar#411]
Hello!

So i want to a key "E" to send the packet 22 to multiogar from my client.
Does anyone here know how to add the code??

the code:
`.onkeydown = function(b) {
switch (b.keyCode) {
case 69:
send(22)
break;`

**Comment by MastaCoder** [comment]
1. This is MultiOgar, the server so we can't help with clients.
2. How are we supposed to know when we don't even know what the client is and how it looks?

**Comment by CalvinDKL** [comment]
Does it matter the client? every client is basically the same, ya know? Coded in html etx. I just need to know how to send a packet. Thats all :)

**Comment by MastaCoder** [comment]
Not really.

**Comment by CalvinDKL** [comment]
Yes, i just need to know if i need to place it over the body for example or in a js etc.
Easy, but anyways i got it now.

**Comment by SimonOrJ** [comment]
You can close the issue by clicking "Close issue" button on the bottom (next to Comment button).
